plutus-tx/test/TH/Spec.hs

```diff
@@ -44,13 +44,16 @@ runPlcCek :: GetProgram a => [a] -> ExceptT SomeException IO EvaluationResultDef
 runPlcCek values = do
      ps <- Haskell.traverse getProgram values
      let p = foldl1 applyProgram ps
-     ExceptT $ try @SomeException $ evaluate $ evaluateCek p
+     let result = evaluateCek p
```

Comment:
`-Wall` doesn't say this is a redundant line?

Reply:
:man_facepalming: